### ISSUE TITLE
Question regarding hosting your template

### ISSUE THREAD

**Issue by jh197033** [ShaunCurtis/Blazr.Demo#1]
I am new to Blazor and came across your template, thank you for your time and effort into creating this for all us newbies.

My question is if I was using your template out of the box, what would I need to do regarding hosting it. Would I need to host the Blazr.Demo.Controllers as a separate IIS instance and then the Blazr.Demo.Server.Web and again another instance of WASM.Web?

**Comment by ShaunCurtis** [comment]
The WASM.Web project is a standard AspNetCore web project.  It runs all the controllers.  See the relevant sections in `Program`

```
services.AddAppWASMServerServices();
services.AddControllers().PartManager.ApplicationParts.Add(new AssemblyPart(typeof(Blazr.Demo.Controllers.WeatherForecastController).Assembly));
```

and

```
app.MapRazorPages();
app.MapControllers();
app.MapFallbackToFile("index.html");
```

Also see this post on WASM hosting - [The-Blazor-WASM-Hosted-Project](https://shauncurtis.github.io/posts/The-Blazor-WASM-Hosted-Project.html)

**Comment by jh197033** [comment]
So does that mean I need to host both the client and the server in the IIS?

example
server.domain.com (all the controller api's)
client.domain.com (the WASM project)

**Comment by ShaunCurtis** [comment]
No, ONE server project.  WASM is just a set of files available through the Server project.  For deployment do a deploy from the Server project only.  All the files needed will be deployed.

**Comment by jh197033** [comment]
Ok I tested this today after setting up an azure account and published the vanilla blazor WebAssembly with asp.net core hosted and PWA option selected and it worked. But Now the confusion gets worst, if the Server project is acting like the API controller then why have I seen other examples of users creating a Client Only with PWA selected and then create a ASP.NET Core Web API project and call the api's from there and not select the basic ASP.NET Core hosted option. Is it something to do with code distribution?
